src/Profile/ProfileSourceLocalFs.php

```diff
@@ -14,53 +14,55 @@ class ProfileSourceLocalFs implements ProfileSourceInterface {
   /**
    * {@inheritdoc}
    */
-  public function getName()
-  {
-    return 'localfs';
-  }
+    public function getName()
+    {
+        return 'localfs';
+    }
 
   /**
    * {@inheritdoc}
    */
-  public function getList()
-  {
-    $cache = Container::cache($this->getName())->getItem('profiles');
-    if ($cache->isHit()) {
-      return $cache->get();
-    }
-    $finder = new Finder();
-    $finder->files()
-      ->in('.')
-      ->name('*.profile.yml');
+    public function getList()
+    {
+        $cache = Container::cache($this->getName())->getItem('profiles');
+        if ($cache->isHit()) {
+            return $cache->get();
+        }
+
+        $directories = array('.', getenv('HOME') . '/.drutiny/');
```

Comment:
I think this is the only change your suggesting here which seems fine but I think your text editor has changed the spacing from 2 spaces to 4 so this file looks like it has a lot more change in it. Is it possible to do get just this change and not the spacing changes. If spacing should be moved to 4 spaces then that should be a seperate PR for the entire project.

Reply:
Ah dang it, thanks for pointing that out Josh! will fix, I think I have a quirky setting in Sublime